Processing/AppDelegate.swift

```diff
@@ -27,6 +32,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationDidEnterBackground(_ application: UIApplication) {
         // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
         // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
+        NotificationCenter.default.post("saveCode")
```

Comment:
We should probably start to put (string) constants into a separate file so we can reference them with code instead of typing the string each time.